### PR TITLE
Fixes for na_ontap_cluster_peer

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Allow for more than 1 source and destination address

* source_intercluster_lif (string) is now source_intercluster_lifs (list)
* dest_intercluster_lif (string) is now dest_intercluster_lifs(list)

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
* na_ontap_cluster_peer

##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0 (netapp-1043 ecc1885692) last updated 2018/11/07 14:58:48 (GMT -700)
  config file = None
  configured module search path = [u'/Users/chrisarchibald/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chrisarchibald/Code/github/ansible/lib/ansible
  executable location = /Users/chrisarchibald/Code/github/ansible/bin/ansible
  python version = 2.7.12 (default, Oct 11 2016, 05:20:59) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/porting_guides/porting_guide_2.8.rst [hunk 1]
```diff
@@ -110,6 +110,9 @@ Noteworthy module changes
 
 * The ``vmware_local_role_facts`` module now returns a list of dicts instead of a dict of dicts for role information.
 
+* The ``na_ontap_cluster_peer`` module has replace ``source_intercluster_lif`` and ``dest_intercluster_lif`` string options with
```
Comment (acozine) [inline docs/docsite/rst/porting_guides/porting_guide_2.8.rst]: typo here - should be "has replaced" not "has replace"

**Review by lonico — APPROVED** [review]

**Comment by acozine** [comment]
@carchi8py looks good from a docs point of view.

**Comment by gundalow** [comment]
@carchi8py Can you please rebase then we can merge https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

**Comment by acozine** [comment]
Thanks @carchi8py.

**Pull request merged**